### DISCUSSION TITLE
How to use the debug features, is here some footage (somewhere??)

### DISCUSSION THREAD

**Discussion by HansNieuwStadt** [TabularEditor/TabularEditor3#457] [🙋♀️TE3 Q&A]
Hey,

Happy with this super-tool!
 
I'm relatively new to Power BI and that's one of the reasons I bought this beautiful tool (I don't call it an "editor" because it is so much more in my opinion).  

I bought this tool with one reason in mind: I know I have to learn a lot of this "eco-system" Power BI, DAX, M, modeling in general etc... So, this tool (Tab-Editor) is, for now, too complicated, but why first "learning to ride a bicycle while I need to fly "an airplane" later? So, I start directly with "the airplane" and find a good instructor to teach me to fly ". 

**That brings me to my question: It there somewhere any good footage about "learning how to use the Tabular debugger in proper manner?" **

regards  Hans

**Comment by otykier** [comment]
Hi Hans,
Glad you like the tool.
We’re working on some recordings. In the mean time, make sure to check out this article: https://docs.tabulareditor.com/te3/features/dax-debugger.html

**Comment by HansNieuwStadt** [comment]
Hi Darren

Thank you for answering my request. I will follow you're
recommendation. 
Keep going with the good work!!! and sincerely thanks!!!!!  

_________________________
Hans de Cocq
Op de Kasselemeier 40
Nieuwstadt
Mobile +31650620777

On March 7, 2022, Darren Gosbell ***@***.***> wrote:
> Hi Hans,
>  Glad you like the tool.
>  We’re working on some recordings. In the mean time, make sure to
> check out this article:
> <https://docs.tabulareditor.com/te3/features/dax-debugger.html>
>

—
Reply to this email directly, view it on GitHub
<https://github.com/TabularEditor/TabularEditor3/discussions/457#discussioncomment-
2309694>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AXEJ4FS2KZS3EW6ZK5X2BATU6YR7TANCNFSM5QDUN7ZQ>.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-
email&mt=8&pt=524675> or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-
email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you authored the thread.Message ID:
<TabularEditor/TabularEditor3/repo-
***@***.***>
